portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -2757,9 +2757,9 @@ $.extend(true, doaj, {
 
                 var subjects = "";
                 if (edges.hasProp(resultobj, "index.classification_paths") && resultobj.index.classification_paths.length > 0) {
-                    subjects = '<h4>Journal subjects</h4><ul>';
-                    subjects += "<li>" + resultobj.index.classification_paths.join("<br>") + "</li>";
-                    subjects += '</ul>';
+                    subjects = '<h4>Journal subjects</h4><ul class="inlined-list">';
+                    subjects += "<li>" + resultobj.index.classification_paths.join(",&nbsp;</li><li>") + "</li>";
+                    keywords += '</ul>';
```

Comment:
This has to stay `subject`. Otherwise, when keywords do not exist, the subjects are not displayed (due to the lack of a closing tag).

Reply:
Oops, that was a copy-pasting error ;p